third_party/rules_python/BUILD

```diff
@@ -0,0 +1,5 @@
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]) + ["//third_party/rules_python/python:srcs"],
+    visibility = ["//visibility:public"],
```

Comment:
Limit the visibility to just //third_party:srcs. Same for python:srcs

Reply:
Done, let's see how CI likes it